<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optimize_bowers_Dialog</class>
 <widget class="QDialog" name="optimize_bowers_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Optimize Bowers</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Wells</string>
     </property>
     <widget class="QListWidget" name="well_listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>131</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Velocity:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="velocity_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pressure:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>OBP:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="obp_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="scatter_Button">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>380</y>
        <width>65</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Scatter</string>
      </property>
     </widget>
     <widget class="QListWidget" name="pres_listWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>270</y>
        <width>131</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Optimize</string>
     </property>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>141</width>
        <height>80</height>
       </rect>
      </property>
      <property name="title">
       <string>Save</string>
      </property>
      <widget class="QPushButton" name="save_Button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>65</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>N</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>141</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Loading</string>
      </property>
      <widget class="QPushButton" name="fit_loading_Button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Fit</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="a_lineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>98</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>20</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>A:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>40</y>
         <width>20</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>B:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="b_lineEdit">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0.86</string>
       </property>
      </widget>
      <widget class="QPushButton" name="draw_loading_Button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Draw</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clear_loading_Button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="loading_textEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>141</width>
        <height>171</height>
       </rect>
      </property>
      <property name="title">
       <string>Unloading</string>
      </property>
      <widget class="QPushButton" name="fit_unloading_Button">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Fit</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>U:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="vmax_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>4800</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="u_lineEdit">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>0.86</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Vmax:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="draw_unloading_Button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Draw</string>
       </property>
      </widget>
      <widget class="QPushButton" name="clear_unloading_Button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>140</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="unloading_textEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
